schema/schema.cue

```diff
@@ -62,7 +62,7 @@ mappings!: [
 		rules!: [
 			...{
 				input!: {
-					product_category_id: string & =~#category_gid_regex
+					product_category_id: string
```

Comment:
This is fine for now until we add a means for the schema to be defined for each input/output type 🤔